### DISCUSSION TITLE
New components added to existing MSI failing to install as part of upgrade

### DISCUSSION THREAD

**Discussion by AndySmiffy** [wixtoolset/issues#8209] [Questions] [answered]
Firstly a quick thanks - we have been using WiX since 2013 and have found it to be pretty stable and has saved our install team countless hours. In fact it's this stability which has stumped me somewhat as I'm obviously missing something for what should be a simple problem..

In a nutshell newly added files in components are not installing as part of an upgrade, though they ARE installed fine for a new installation. 
I've previously 
a) simply added the new file element to the existing component
b) created a new component in the existing feature, and finally / currently
c) Adding a new feature & component for the new file element as per the below snippet, but neither is working. 

      <Component Id="cmpNewtonSoft" Guid="565D8C7F-75B9-427B-AD6A-0E10F0B16FC5">
          <File Source="..\..\External References\Newtonsoft.json\13.0.1\Newtonsoft.Json.dll"/>
        </Component>

      <Feature Id="fNewtonSoft" Level="1">
        <ComponentRef Id="cmpNewtonSoft"/>
      </Feature>

    </Fragment>

If anyone has any ideas as to what would be causing this I'd be all ears - thanks.
WiX 3.14 using the Wix Toolset Visual Studio extension. It's in an MSI that is part of a bootstrapper

**Comment by chrpai** [comment]
What kind of upgrade?  What does the MSI log show?

**Reply by AndySmiffy** [reply]
Hi Christopher. Its a major upgrade. Unfortunately the upgrade has for some reason stopped logging which is something of a pain, and has only come to light when trying to see whats going on with this issue. 
If there's a list of things that would stop an upgrade logging this would be the best step in finding what's going on, but so far my google-fu has failed me in finding this. NB A clean install does create the log files, "only" the upgrade doesn't. 
I've previously added verbose logging to the MSI project to see if this changed the situation but alas not.

    <Property Id="MSILOGGING" Value="v" />

**Reply by chrpai** [reply, marked as answer]
Property names in MSI are case sensitive.  The property you are looking for is `MsiLogging` and you'll want to set it to `voicewarmup`.  You can also launch your msi with logging  with a command like `msiexec /i foo.msi /l*v path_to_log`
